.github/workflows/announce-release.yml

```
@@ -0,0 +1,35 @@
+name: Announce Release on Mastodon
+
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+permissions:
+  pull-requests: read
+
+jobs:
+  post_to_mastodon:
+    if: |
+      github.event.pull_request.merged == true &&
+      (contains(github.event.pull_request.title, 'release') || contains(github.event.pull_request.title, ' v') || contains(github.event.pull_request.title, 'version'))
```

Maybe make this more specific, locking it to the exact title that release-please creates:
```suggestion
    if: contains(github.event.pull_request.title, 'chore(main): release')
```
Also, I think the `pull_request_title` never contains a `v` or a `version`, right? And with the different targeting of the workflow I suggested above, I think there is also no need for the `github.event.pull_request.merged` check.